src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/EventsController.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Repository;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Altinn.Platform.Events.Controllers
+{
+    /// <summary>
+    /// Provides operations for handling events
+    /// </summary>
+    [Route("events/api/v1/events")]
+    [ApiController]
+    public class EventsController : ControllerBase
+    {
+        private readonly IEventsRepository repository;
```

Comment:
We usually name the private variables in the platform with a _ to begin with. e.g. _repository.

Reply:
Hmm, I see this isn't done for applicationsController where you copied the code, but I would still like to see it here. :)

Reply:
Done